### DISCUSSION TITLE
【求助】六分街的空气墙

### DISCUSSION THREAD

**Discussion by beibei-b** [share121/inter-knot#1586] [其他]
我是一名六分街的普通居民，今天我在出门散步的时候，到了一个路口突然碰到了一堵空气墙！无论我怎么尝试，都有一面看不见的墙在阻止我往前走！
更恐怖的是，只有我一个人有这种情况，周围的街坊邻居们来来往往，只有我过不去那个路口！
有没有大佬看看这是什么问题啊！我是不是吃了啥不该吃的
![KBGJ_1723378597863_edit_50720306094295](https://github.com/user-attachments/assets/34e5f729-a913-415d-b0f1-f14ea9ef968b)
![KBGJ_1723378868665](https://github.com/user-attachments/assets/e9edee72-e26f-4384-b2c5-36eb1cfd33c3)

**Comment by khiqwq** [comment]
应该是你的等级不够，够了就开放了（￣︶￣）↗

**Comment by ZhuYuan-148** [comment]
治安局已抵达现场。

**Comment by sfhjhsdfg** [comment]
我听一些盗洞客~~朋友~~说他们在六分街周围感觉像是进了空洞一样，怀疑一整个六分处在空洞和现实的叠加态，光映广场好像也是的

**Comment by gaojiaobei** [comment]
莫非我们的世界是游戏世界？我们都是游戏中的人物？

**Comment by YoKiiiiiiiiiiiii** [comment]
某不知名音：前面的区域以后再来探索吧🎵

**Comment by kaijinti** [comment]
新艾莉都，神奇吧
